examples/audio/pesq.py

```diff
@@ -0,0 +1,114 @@
+"""
+PESQ Metric Calculation for Speech Enhancement
+==============================================
+
+In this notebook, we will calculate the Perceptual Evaluation of Speech Quality (PESQ) score
+to assess the improvement in speech quality after applying a basic noise reduction technique.
+
+PESQ is widely used in speech enhancement, telecommunications, and VoIP to evaluate the
+perceived quality of speech signals.
```

Comment:
Imagine you’re on a noisy street, trying to have a phone call. The technology behind the scenes aims to clean up your voice and make it sound clearer on the other end. But how do engineers measure that improvement? This is where PESQ comes in. In this notebook, we will simulate a similar scenario, applying a simple noise reduction technique and using the PESQ score to evaluate how much the speech quality improves.